### PR TITLE
[18.0][MIG] account_statement_base: Migration to 18.0

### PR DESCRIPTION
**Pull request by xaviedoanhduy**
### change in v18

- `checked` was replace `to_checked` fields in commit: https://github.com/odoo/odoo/commit/242e1a04fc97131ccc28905de58011127f078e30

### CODE REVIEW

File: account_statement_base/views/account_bank_statement_line.xml [hunk 1]
```diff
@@ -145,9 +145,9 @@
                 />
                 <separator />
                 <filter
-                    name="to_check"
+                    name="checked"
                     string="To check"
-                    domain="[('to_check', '=', True)]"
+                    domain="[('checked', '=', True)]"
                 />
```
Comment (mmequignon) [inline account_statement_base/views/account_bank_statement_line.xml]: Why not keeping this filter ?
```suggestion
                <filter
                    name="to_check"
                    string="To check"
                    domain="[('checked', '=', False)]"
                />
```

**Review by mmequignon — APPROVED** [review]

File: account_statement_base/views/account_bank_statement_line.xml (hunk 1 above)
Comment (xaviedoanhduy) [reply account_statement_base/views/account_bank_statement_line.xml]: > checked was replace to_checked fields in commit: https://github.com/odoo/odoo/commit/242e1a04fc97131ccc28905de58011127f078e30

Comment (xaviedoanhduy) [reply account_statement_base/views/account_bank_statement_line.xml]: hi @mmequignon,
I have keeped it. thanks

**Comment by victoralmau** [comment]
Please cherry-pick https://github.com/OCA/account-reconcile/pull/751 to commit history

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by victoralmau** [comment]
Please cherry-pick https://github.com/OCA/account-reconcile/pull/752 to commit history

**Comment by xaviedoanhduy** [comment]
hi @victoralmau, Done!

**Comment by victoralmau** [comment]
Thanks, I think you forgot the commit _[FIX] account_statement_base: Remove wrong field label_ from https://github.com/OCA/account-reconcile/pull/751

File: account_statement_base/views/account_bank_statement_line.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
                     <field name="company_id" invisible="1" />
                     <field name="currency_id" invisible="1" />
                     <field name="country_code" invisible="1" />
-                    <field5c1485a3 name="state" invisible="1" />
```
Comment (xaviedoanhduy) [inline account_statement_base/views/account_bank_statement_line.xml]: hi @victoralmau, during migration state, I fixed it

**Comment by xaviedoanhduy** [comment]
> Thanks, I think you forgot the commit _[FIX] account_statement_base: Remove wrong field label_ from #751

during the migration state, i fixed it

File: account_statement_base/views/account_bank_statement.xml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        </field>
+    </record>
+
+    <!-- Add form to view mode. -->
+    <record id="account.action_bank_statement_tree" model="ir.actions.act_window">
+        <field name="res_model">account.bank.statement</field>
+        <field name="view_mode">list,form,pivot,graph</field>
+    </record>
+    <record id="account.action_view_bank_statement_tree" model="ir.actions.act_window">
+        <field name="view_mode">tree,form,pivot,graph</field>
```
Comment (victoralmau) [inline account_statement_base/views/account_bank_statement.xml]:
```suggestion
        <field name="view_mode">list,form,pivot,graph</field>
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
Have you functionally tested the module? I'm not sure why but if you install `account_statement_base` and add the corresponding permission it doesn't show the Dashboard menu in Invoicing, can you check it?

File: account_statement_base/views/account_bank_statement.xml (hunk 3 above)
Comment (xaviedoanhduy) [reply account_statement_base/views/account_bank_statement.xml]: have conditions to see this action in [here](https://github.com/odoo/odoo/blob/33e32433528025625efa2b87839b1f6fd3bb1f9b/addons/account/views/account_journal_dashboard_view.xml#L70) 
image

**Comment by xaviedoanhduy** [comment]
> Have you functionally tested the module? I'm not sure why but if you install `account_statement_base` and add the corresponding permission it doesn't show the Dashboard menu in Invoicing, can you check it?

hi @victoralmau, have conditions to see this action in [here](https://github.com/odoo/odoo/blob/33e32433528025625efa2b87839b1f6fd3bb1f9b/addons/account/views/account_journal_dashboard_view.xml#L70) 

image

**Review by victoralmau — APPROVED** [review]
Code and functional review OK

**Review by pedrobaeza — APPROVED** [review]
/ocabot migration account_statement_base
/ocabot merge nobump

**Pull request merged**